Examples/UIExplorer/NavigatorIOSColorsExample.js

```diff
@@ -66,6 +66,7 @@ var NavigatorIOSColors = React.createClass({
         tintColor="#FFFFFF"
         barTintColor="#183E63"
         titleTextColor="#FFFFFF"
+        fontFamily="Helvetica"
```

Comment:
Just tested, it appears to only affect the title. I must have been playing with some other title.

I'll get that changed and update the PR.